alot/crypto.py

```diff
@@ -196,17 +169,18 @@ def verify_detached(message, signature):
 
     :param message: the message as `str`
     :param signature: a `str` containing an OpenPGP signature
-    :returns: a list of :class:`gpgme.Signature`
+    :returns: a list of :class:`gpg.results.Signature`
     :raises: :class:`~alot.errors.GPGProblem` if the verification fails
     '''
-    message_data = StringIO(message)
-    signature_data = StringIO(signature)
-    ctx = gpgme.Context()
-
-    status = ctx.verify(signature_data, message_data, None)
-    if isinstance(status[0].status, gpgme.GpgmeError):
-        raise GPGProblem(status[0].status.message, code=status[0].status.code)
-    return status
+    ctx = gpg.core.Context()
+    try:
+        verify_results = ctx.verify(message, signature)[1]
```

Comment:
look for `pyme` (which is what the gpg bindings were called before getting blessed as the official ones). http://pyme.sourceforge.net/doc/pyme/pyme.core.html

Honestly the documentation for the gpg bindings is sparse, and usually they just say "read what the C code in gpgme does", which is pretty frustrating. there are some docstrings, so pydoc can help too.